includes/definitions/terra.yaml

```diff
@@ -0,0 +1,10 @@
+os: terra
```

Comment:
It covers channel processor devices (DVB-T to IP, DVB-S to DVB-C etc, index 5-10 and 14-18).
All have custom embbeded OS (different for each model), I made tests only for sti410 (9) and sti440 (18). Each device has its own MIBs available after purchase. Only version and serial number are under the same OID.

It doesn't cover products with index:
1 - OD120 - optical receiver - discontinued
2 - MO411 - optical transmitter (other purpose)
3 - MO4x8 - optical transmitter (other purpose - not DVB processor)
4 - UC412 - can't find any info
11 - S2C16 - Similar GUI to sti410, should also work
12 - OT001M - can't find any info
13 - S2C16P - Similar GUI to sti410, should also work